### PR TITLE
Dscs 361

### PR DESCRIPTION
**Pull request by shuangzhou**
This request provides the api to order copies of an audio work for oral history sound upload.

### CODE REVIEW

**Comment by shuangzhou** [comment]
@ninhnguyen @scoen could you please review this change?  Thanks.

File: src/amberdb/enums/CopyRole.java [hunk 1]
```diff
@@ -1,79 +1,79 @@
...
-    PAPER_TRANSCRIPT("tp", "Paper transcript", "No"),
-    ELECTRONIC_SUMMARY("se", "Electronic Summary", "No"),
-    ELECTRONIC_TRANSCRIPT("te", "Electronic transcript", "No"),
-    TIME_CODED_SUMMARY("sc", "Time coded Summary", "Yes"),
-    TIME_CODED_TRANSCRIPT_COPY("tc", "Time coded transcript", "Yes"),
-    SUMMARY_COPY("s", "Other Summary", "No"),
-    RTF_TRANSCRIPT("tt", "rtf transcript", "No"),
-    TRANSCRIPT_COPY("tr", "Other Transcript", "No"),
-    PRODUCTION_MATERIAL("pm", "Production Material");
+    ORIGINAL_COPY("o", "Original",0),
```
Comment (AaronCMuller) [inline src/amberdb/enums/CopyRole.java]: Relying on ordinals is generally a poor way to order enums.

See here... http://allenlsy.com/NOTES-of-Effective-Java-5/#31.-use-instance-fields-instead-of-ordinals

Comment (shuangzhou) [inline src/amberdb/enums/CopyRole.java]: @AaronCMuller I can see what you mean.  I hope there might be a better solution later, at the moment, I'm creating an instance field to explicitly specify the relative order for copies.  It works for the moment.

Comment (AaronCMuller) [inline src/amberdb/enums/CopyRole.java]: Sorry, I didn't see that other stuff.

Ignore my comment.

**Pull request merged**